tests/suites/1_stateful/08_select_stage/08_00_parquet/08_00_00_basic.sh

```diff
@@ -35,6 +35,16 @@ echo "drop stage if exists s3;" | $MYSQL_CLIENT_CONNECT
 echo "create stage s3 url = '${DATADIR}' FILE_FORMAT = (type = CSV);"  | $MYSQL_CLIENT_CONNECT
 echo "set use_parquet2 = ${USE_PARQUET2} ; select * from @s3 (FILE_FORMAT => 'PARQUET');" | $MYSQL_CLIENT_CONNECT
 
+
```

Comment:
for read only tests.
it would be easier to maintain if use sqllogic test.
under suit stage_parquet